### ISSUE TITLE
Turn `workshop` and `startercode` branches into Git Submodules instead

### ISSUE THREAD

**Issue by nczhu** [substrate-developer-hub/utxo-workshop#61]
So branch version control becomes a bit easier. 
- Workshop: points to the latest commit
- Startercode: points to Nicole's commit before the PoW changes

@JoshOrndorff I think this will make this repo less confusing to navigate. We can turn this master repo into `Substrate Bitcoin` and have it be the full implementation / reference for teams that want to build Bitcoin chains. 

I feel like keeping this repo as a workshop devalues how functional and comprehensive it is.

**Comment by JoshOrndorff** [comment]
I agree separating the utxo workshop from the entire bitcoin-like chain makes sense for sure.

I don't yet fully understand where the line will be drawn though. Would some code still live in this repo? If yes, would it include the utxo pallet? Would the Substrate Bitcoin repo also include that code, or depend o nit from this repo?

What if instead of getting its own repo, the bitcoin stuff became a recipe. Would that still work with your plan here?

**Comment by nczhu** [comment]
mm yeah... im overcomplicating things.  Your suggestion of turning the PoW stuff into a recipe sounds great. 

Let's merge your latest PR to this repo, once it's ready, and just keep the current repo/branch structure. I realize that keeping two separate repos, one for the full impl and one for the workshop boilerplates, doesn't simplify things all that much...
